src/include/miopen/cat/problem_description.hpp

```diff
@@ -111,9 +109,7 @@ struct ProblemDescription : ProblemDescriptionBase
     {
         if((dim < 0) || (dim > yDesc.GetLengths().size()))
         {
-            MIOPEN_THROW(
-                miopenStatusBadParm,
-                "CatForward: is greater than 0 and less than or equal tensor dimension length.");
+            return false;
```

Comment:
Ditto

Reply:
changed it.

Reply:
See https://github.com/ROCm/MIOpen/pull/2562#discussion_r1437751964